docs/endpoints/activity.md

```diff
@@ -58,7 +58,7 @@ Example Response:
               "caption": "Captioning captions",
             },
             "tagged": [
-              "good-photo",
+              "good-submission",
               "good-quote"
             ],
             "reactions": [
```

Comment:
This is a sidenote but I forgot to update the documentation endpoints when I was [restructuring files in Rogue](https://github.com/DoSomething/rogue/pull/654) ! I'll do that in quickly in another little PR :)